crm_claim_rma_number/model/crm_claim_rma.py

```diff
@@ -0,0 +1,125 @@
+# -*- encoding: utf-8 -*-
+###############################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#    Copyright (C) OpenERP Venezuela (<http://www.vauxoo.com>).
+#    All Rights Reserved
+# ############ Credits ########################################################
+#    Coded by: Yanina Aular <yani@vauxoo.com>
+#    Planified by: Yanina Aular <yani@vauxoo.com>
+#    Audited by: Nhomar Hernandez <nhomar@vauxoo.com>
+###############################################################################
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class claim_line(models.Model):
+
+    _inherit = 'claim.line'
+
+    @api.model
+    def _get_sequence_number(self):
+        """
+        @return the value of the secuence for the number field in the
+        claim.line model.
+        """
+        seq_obj = self.env['ir.sequence']
+        return seq_obj.get('claim.line')
+
+    number = fields.Char(
+        readonly=True,
+        default='/',
+        help='Claim Line Identification Number')
+
+    # Field "number" is assigned by default with "/"
+    # then this constraint ever is broken
+    # _sql_constraints = [
+    #     ('number_uniq', 'unique(number, company_id)',
+    #         'Internal RMA number must be unique per Company!'),
+    # ]
+
+    @api.model
+    def create(self, vals):
+        """
+        @return wirte the identify number once the claim line is create.
+        """
+        vals = vals or {}
+        if ('number' not in vals) or (vals.get('number', False) == '/'):
+            vals['number'] = self._get_sequence_number()
+        res = super(claim_line, self).create(vals)
+        return res
+
+    @api.multi
+    def name_get(self):
```

Comment:
Use `@api.one` and the looping and putting results in the list is done automatically